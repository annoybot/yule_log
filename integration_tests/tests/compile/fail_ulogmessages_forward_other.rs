use yule_log::{ULogMessages, ULogData};

#[derive(ULogData)]
pub struct A { value: u64 }

#[derive(ULogMessages)]
pub enum LoggedMessages {
    A(A, A),
}

fn main() {}
