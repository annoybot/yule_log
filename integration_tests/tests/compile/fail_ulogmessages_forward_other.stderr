error: Enum variants must be tuple variants containing exactly one struct type.
 --> tests/compile/fail_ulogmessages_forward_other.rs:8:5
  |
8 |     A(A, A),
  |     ^^^^^^^
